app/gui/controller/double-representation/src/project.rs

```diff
@@ -76,6 +76,11 @@ impl QualifiedName {
         }
     }
 
+    /// Return the qualified name of the "Standard.Base" library project.
+    pub fn of_standard_base_library() -> Self {
```

Comment:
We've been never using such naming convention (starting functions with "of"). I've never seen it in any Rust lib neither - do we use it somewhere else? If not, let's change it to a function name without such a prefix.